libdnf/repo/Repo.cpp

```diff
@@ -33,12 +33,24 @@
 #define DIGITS "0123456789"
 #define REPOID_CHARS ASCII_LETTERS DIGITS "-_.:"
 
+#define MD_TYPE_PRIMARY "primary"
+#define MD_TYPE_FILELISTS "filelists"
+#define MD_TYPE_PRESTODELTA "prestodelta"
+#define MD_TYPE_GROUP_GZ "group_gz"
+#define MD_TYPE_GROUP "group"
+#define MD_TYPE_UPDATEINFO "updateinfo"
+#define MD_TYPE_MODULES "modules"
```

Comment:
We also support downloading AppStream data, don't we? We should have that in here...

Reply:
These constants are not supposed to be a complete list of the supported metadata types. I just did not like to have a lot of string literals hanging around.